Documentation/ART_NOTES.md

```diff
@@ -0,0 +1,119 @@
+  >* Higher values in your settings will make baking take longer
+    >* It's recommended to start with very low values for your settings to make baking fast and lighting iteration easy. Then, when you have something you like, you can bump up these settings for a slower, but better quality bake 
+    >* For example, start with a low Lightmap Resolution, like 1, for iteration. Then, bump it up to 6 or 8 for a higher quality bake 
+
+ <br />       
+    
+* You can hover over each setting and a tooltip will descirbe what it does <br /><br />
+
+* Directional Mode toggles to whether or not normal maps will influence your baked lighting. Changing your settings to Directional will generate a better quality bake, but is more costly. When baking, additional textures, called directional maps, will be generated for every baked lightmap texture. For example, if you have three lightmaps, you will have three directional maps.  <br /><br />
+
+* In the lighting panel, there is a checkbox that allows Unity to auto-generate lighting whenever your lighting changes, a model is moved, a lighting setting is changed, etc.. This can be useful for quick lighting iteration
+ 
+  ![AutoGenerate](Images/AutoGenerateLighting.png) <br /><br />
+
+* To clear baking data for your scene: click the arrow on the Generate Lighting button. From the drop down menu, click Clear Baked Data <br /><br />
+
+* Multiple scenes can be baked at once by loading them together (click and drag scenes into the heirarchy), and generating lighting. Overlapping geometry in your scenes scenes can cause baking artifacts though, so only bake them together if they're not overlapping <br /><br />
+
+* Meshes must be set to contribute to baked Global Illumination to receive baked lighting 
+  
+  ![MeshSettings](Images/MeshSettings.png) <br /><br />
+
+* Reflection probes can also be baked to save performance <br /><br />
+
+## **Baking for Boss Room**
+
+We've set up some prefabs and scenes to make baking easier.
+
+* Thre is a lighting settings asset called FastBaking (Assets > BossRoom > Scenes > Bossroom) that has pre set up settings for a quick, but decent bake for Boss Room.
+
+* There is a light probe group prefab called BossroomLightprobes (Assets > Bossroom > Prefabs > Dungeon) to be used with baked lighting. Light probes store baked lighting from your scene to be used on objects moving through the scene. Check out the sources below for more info on light probes <br /><br />
+
+### **Lighting set up for the MainMenu, CharSelect, and PostGame Scenes:**
+
+* To bake these scenes, open them and navigate to the CharSel_Background Prefab in the scene. In the prefab, there is are two other prefabs called "EnvLighting_Realtime" and "EnvLighting_ForBaking_" with lights set up for realtime, and lights set up for baking. Leave the Realtime prefab enabled if you're using realtime lighting, otherwise, disable it and enable the Baking prefab when baking lighting.
```

Comment:
```suggestion
* To bake these scenes, open them and navigate to the CharSel_Background Prefab in the scene. In the prefab, there are two other prefabs called "EnvLighting_Realtime" and "EnvLighting_ForBaking_" with lights set up for realtime, and lights set up for baking. Leave the Realtime prefab enabled if you're using realtime lighting, otherwise, disable it and enable the Baking prefab when baking lighting.
```